src/inverseRobot.js

```diff
@@ -7,7 +7,17 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const truePosition = {};
+
+  for (const key in robot) {
+    if (truePosition.hasOwnProperty(robot[key])) {
+      return null;
+    }
+
+    truePosition[robot[key]] = key;
+  }
+
+  return truePosition;
```

Comment:
The function correctly returns the 'truePosition' object if no duplicate values are found in the input 'robot' object, adhering to the task requirements.